chamge/amqp-edge-backend.c

```diff
@@ -268,11 +216,20 @@ _amqp_rpc_request (amqp_connection_state_t amqp_conn, guint channel,
 
   g_debug ("declare a queue for reply : %s", (gchar *) amqp_reply_queue.bytes);
 
-  /* send the message */
-  if (_amqp_property_set (&amqp_props, channel,
-          (gchar *) amqp_reply_queue.bytes) != CHAMGE_RETURN_OK) {
-    g_error ("create property failed.");
-    return NULL;
+  /* property setting to send rpc request */
+  {
+    memset (&amqp_props, 0, sizeof (amqp_basic_properties_t));
```

Comment:
This will not be required if it is declared with `{ 0 }`

```
  amqp_basic_properties_t amqp_props = { 0 };
```

Reply:
Thanks.